src/user-icon/user-icon.scss

```diff
@@ -4,57 +4,47 @@
 @import '../style/index';
 @import '../text/mixins';
 
-@mixin iui-user-icons($size: $iui-icons-xl) {
+@mixin iui-user-icon {
   @include iui-reset;
-
   @include iui-font-family;
   display: inline-block;
   position: relative;
-  width: $size;
-  height: $size;
   margin: 0;
   user-select: none;
 
-  > .iui-user-icons-stroke {
+  > .iui-stroke {
     display: block;
     position: absolute;
     z-index: 2;
```

Comment:
I know this PR is primarily for shortening classenames but we need to think about removing z-indices in the future.

Reply:
I just looked into it and realized it was a very simple fix, only involved re-arranging the HTML.  Correct order from top to bottom is:
1. `abbr`
2. `img`
3. `.iui-stroke`
4. `.iui-status`